articles/spring-cloud/quickstart-deploy-enterprise.md

```diff
@@ -0,0 +1,374 @@
+---
+title: "Quickstart - Build and deploy apps to Azure Spring Cloud Enterprise Tier"
+description: Describes app deployment to Azure Spring Cloud Enterprise tier.
+author: maly7
+ms.author: 
+ms.service: spring-cloud
+ms.topic: quickstart
+ms.date: 
+ms.custom: 
+---
+# Quickstart: Build and deploy apps to Azure Spring Cloud using the Enterprise Tier
+
+**This article applies to:** ❌ Basic/Standard tier ✔️ Enterprise tier
+
+This quickstart shows you how to build and deploy applications to Azure Spring Cloud using the Enterprise tier.
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- A license for Azure Spring Cloud Enterprise Tier. For more information, see [View Azure Spring Cloud Enterprise Tier Offer in Azure Marketplace](./how-to-enterprise-marketplace-offer.md).
+- [The Azure CLI version 2.0.67 or higher](/cli/azure/install-azure-cli).
+- [Git](https://git-scm.com/).
+- [jq](https://stedolan.github.io/jq/download/)
+- [!INCLUDE [install-enterprise-extension](includes/install-enterprise-extension.md)]
+
+## Download the sample app
+
+Use a local terminal for the following steps.
+
+1. Create a new folder and clone the sample app repository.
+
+   ```bash
+   mkdir source-code
+   ```
+
+   ```bash
+   cd source-code
+   ```
+   ```bash
+   git clone https://github.com/Azure-Samples/acme-fitness-store
+   ```
+
+1. Navigate into the repository directory.
+
+   ```bash
+   cd acme-fitness-store
+   ```
```

Comment:
Hi @maly7 - I see that you've copied this from an existing article, but I recommend a change here anyway (and we'll update the existing article at some point, too). The snippets should not be in separate blocks. Also, we don't need to direct the user to create the parent folder. (BTW, be sure the backticks are correct - they might be dropped due to the suggestion syntax's usage of backticks.)

```suggestion
Use the following commands to download the sample:

```bash
git clone https://github.com/Azure-Samples/acme-fitness-store
cd acme-fitness-store
```
```